taca/storage/storage.py

```diff
@@ -178,35 +179,56 @@ def cleanup_uppmax(site, days, dry_run=False):
         ## work flow for cleaning up illumina/analysis ##
         projects = [ p for p in os.listdir(root_dir) if re.match(filesystem.PROJECT_RE,p) ]
         list_to_delete = get_closed_projects(projects, pcon, days)
+        ## delete and log
+        for item in list_to_delete:
+            if dry_run:
+                logger.info('Will remove {} from {}'.format(item,root_dir))
+                continue
+            try:
+                shutil.rmtree(os.path.join(root_dir,item))
+                logger.info('Removed project {} from {}'.format(item,root_dir))
+                with open(log_file,'a') as to_log:
+                    to_log.write("{}\t{}\n".format(item,datetime.strftime(datetime.now(),'%Y-%m-%d %H:%M')))
+            except OSError:
+                logger.warn("Could not remove path {} from {}"
+                        .format(item,root_dir))
+                continue
     else:
         ##work flow for cleaning archive ##
-        list_to_delete = []
-        archived_in_swestore = filesystem.list_runs_in_swestore(path=CONFIG.get('cleanup').get('swestore').get('root'), no_ext=True)
         runs = [ r for r in os.listdir(root_dir) if re.match(filesystem.RUN_RE,r) ]
         with filesystem.chdir(root_dir):
             for run in runs:
-                fc_date = run.split('_')[0]
-                if misc.days_old(fc_date) > days:
-                    if run in archived_in_swestore:
-                        list_to_delete.append(run)
+                ## Collect all project path in the run folder
+                all_proj_path = glob("{}/Unaligned_*/Project_*".format(run))
```

Comment:
this `Project_` is knew to me :-D
It is not present in taca demux... can you make this less hardcoded (e.g., add it in the configuration). Same thing for `Unaligned_*`